web/impact/impact/graphql/types/entrepreneur_profile_type.py

```diff
@@ -49,7 +49,8 @@ def resolve_title(self, info, **kwargs):
 
     def resolve_program_roles(self, info, **kwargs):
         user_roles_of_interest = [UserRole.FINALIST, UserRole.ALUM]
-        startup_roles_of_interest = [StartupRole.ENTRANT]
-        startup_roles_of_interest += StartupRole.WINNER_STARTUP_ROLES
+        startup_roles_of_interest = StartupRole.WINNER_STARTUP_ROLES
+        if info.context.user.is_staff:
```

Comment:
We should not be checking the built-in "is_staff", since that only checks whether a user has access to the admin. Instead, use the `is_employee` function which can be imported from `accelerator_abstract.models.base_user_utils`